plugins/mail_from.is_resolvable.js

```diff
@@ -24,7 +24,8 @@ exports.load_ini = function () {
 exports.hook_mail = function(next, connection, params) {
     var plugin    = this;
     var mail_from = params[0];
-    var results   = connection.transaction.results;
+    var txn       = connection.transaction;
+    var results   = txn.results;
```

Comment:
No - I don't think so.  The hooks aren't run in a setImmediate or process.nextTick so this will all be run in the same tick and the plugins.run_next_hook() function has guards in it already, so it wouldn't get this far if that were the case.